src/WidgetBase.ts

```diff
@@ -280,6 +317,7 @@ export class WidgetBase<P extends WidgetProperties = WidgetProperties, C extends
 	 */
 	private afterUpdateCallback(element: Element, projectionOptions: ProjectionOptions, vnodeSelector: string,
 		properties: VNodeProperties, children: VNode[]): void {
+		this._setNode(element, projectionOptions, vnodeSelector, properties, children);
```

Comment:
Why do you need to pass more than the `element` and `properties` as that looks like it's all the `this._setNode` function needs.